common/src/main/java/com/aliyun/emr/rss/common/network/server/ChannelsLimiter.java

```diff
@@ -65,6 +69,9 @@ private void resumeAllChannels() {
   @Override
   public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
     channels.add(ctx.channel());
+    if (isPaused.get()) {
+      ctx.channel().config().setAutoRead(false);
```

Comment:
If `resumeAllChannels` running between `if (isPaused.get()) {` and this line, some channel will never be set to true?

should we add `sync`?

Reply:
Yes, there is a possibility.

Reply:
I re-checked the logic. The possibility is zero. If `resumeAllChannels` running between `if (isPaused.get()) {` ,  resumeAllChannels function will correct the channel's auto read configuration.